docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```diff
@@ -391,7 +424,26 @@ code: |
   tag_expression = ''
 ---
 code: |
-  test_run_output = background_action('run_alkiln', None, tag_expression=tag_expression, project_name=project_name)
+  test_run_output = background_action('run_alkiln', None, tag_expression=tag_expression, project_name=project_name, env_vars=env_vars)
+---
+code: |
+  import os
+  
+  os_env = os.environ.copy()
+  log(os_env, 'console')
+  
+  temp_names = []
+  for var_key in config_vars:
+    if var_key in os_env:
+      temp_names.append(var_key)
+      
+  dangerous_config_var_names = temp_names
+---
+code: |
+  env_vars = config_vars
```

Comment:
If we're not modifying `config_vars` at all, can we just get rid of this intermediate variable?

Reply:
`config_vars` are defined even if there are dangerous var names. `env_vars` are defined only once there are no dangerous names. I know we force the user to restart, but 1. I worry about the interview restarting properly and 2. I worry about us changing the implementation and forgetting to change this too. I do think this needs other changes, though, to really address those issues.

Reply:
I think it's fine too. Thanks for clarifying!